includes/admin/discounts/add-discount.php

```diff
@@ -141,8 +141,8 @@
 					<label for="edd-start"><?php _e( 'Start date', 'easy-digital-downloads' ); ?></label>
 				</th>
 				<td>
-					<input name="start_date" id="edd-start" type="text" value="" class="edd_datepicker"/>
-					<p class="description"><?php _e( 'Enter the start date for this discount code in the format of mm/dd/yyyy. For no start date, leave blank. If entered, the discount can only be used after or on this date.', 'easy-digital-downloads' ); ?></p>
+					<input name="start_date" id="edd-start" type="text" value="" class="edd_datepicker" data-format="<?php echo edd_get_date_picker_format(); ?>" placeholder="<?php echo edd_get_date_picker_format(); ?>" />
```

Comment:
Escape?

Reply:
Good call.

I’ll probably end up with a new function to handle the escape and echo.

Reply:
Done!